lib/FusionInventory/Agent/Task/Inventory/Win32/Users.pm

```diff
@@ -145,12 +157,18 @@ sub _getLastUser {
     }) or die "Can't open HKEY_LOCAL_MACHINE key: $EXTENDED_OS_ERROR";
 
     my $user =
+        encodeFromRegistry($machKey->{"SOFTWARE/Microsoft/Windows/CurrentVersion/Authentication/LogonUI/LastLoggedOnSAMUser"}) ||
         encodeFromRegistry($machKey->{"SOFTWARE/Microsoft/Windows/CurrentVersion/Authentication/LogonUI/LastLoggedOnUser"}) ||
         encodeFromRegistry($machKey->{"SOFTWARE/Microsoft/Windows NT/CurrentVersion/Winlogon/DefaultUserName"});
     return unless $user;
 
+    my ($domain) = $user =~ /^(.*)\\/;
```

Comment:
Yes, I assume the `domain\user` syntax for `LastLoggedOnSAMUser` as this is what we can find anywhere on internet for it (I searched a long time in technet, but I didn't find any official spec...)
So, we need to check if we may have any confusion in the @orthagh 's issue but actually the related client is not around and we can't check what he has in the related registry key.
So I delayed the integration to be sure we are not messing up anything even if I expected to find something like `AD\adelaunay@teclib.com` in registry. Then if this is the case, this is not even consistent with what seems to be reported in `<USERS/>` and maybe I'll search for a stronger way to scan logged, maybe using WTS API I included in a previous PR.